src/api/qualicharge/api/v1/routers/dynamic.py

```diff
@@ -463,3 +462,22 @@ async def create_session_bulk(
         size=len(db_qc_session_ids),
         items=db_qc_session_ids,
     )
+
+
+@router.get("/session/check", status_code=fa_status.HTTP_200_OK, tags=["Session"])
+async def check_session(
+    user: Annotated[User, Security(get_user, scopes=[ScopesEnum.DYNAMIC_CREATE.value])],
```

Comment:
what is the use of `user`? I don't see it then

Reply:
This ensures that the user is authenticated and has proper permissions.